scripts/battlefields/Empyreal_Paradox/dawn.lua

```diff
@@ -66,8 +68,13 @@ function content:onEventFinishBattlefield(player, csid, option, npc)
     for _, ally in pairs(bcnmAllies) do
         ally:resetLocalVars()
         local spawn = ally:getSpawnPos()
+        -- Logic to refrain Prishe from using Daedalus Wing again
+        if ally:getName() == 'Prishe' then
+            ally:setLocalVar('used_item_1', 1)
+        end
         ally:setPos(spawn.x, spawn.y, spawn.z, spawn.rot)
     end
+    
```

Comment:
None of this is needed, I'm pretty sure. You are already gating its usage to once per battlefiled by checking the variable